discord/activity.py

```diff
@@ -186,13 +188,18 @@ class Activity(BaseActivity):
 
         - ``id``: A string representing the party ID.
         - ``size``: A list of up to two integer elements denoting (current_size, maximum_size).
-    buttons: List[:class:`dict`]
-        An list of dictionaries representing custom buttons shown in a rich presence.
+    buttons: List[Union[:class:`dict`, :class:`str`]]
```

Comment:
Maybe have this be 
```
Union[List[:class:`dict`], List[:class:`str`]]
```
to emphasize that it can either be a list of dict or a list of str, not a list of dict and/or str

Reply:
Good idea, done.